WooCommerce/Classes/ViewRelated/LaunchScreen.storyboard

```diff
@@ -25,12 +25,12 @@
                                 </constraints>
                             </imageView>
                         </subviews>
+                        <viewLayoutGuide key="safeArea" id="6Tk-OE-BBY"/>
                         <color key="backgroundColor" name="WooCommercePurple60"/>
                         <constraints>
                             <constraint firstItem="Q7F-qQ-dHf" firstAttribute="centerX" secondItem="Ze5-6b-2t3" secondAttribute="centerX" id="Ilw-Kx-7ga"/>
                             <constraint firstItem="Q7F-qQ-dHf" firstAttribute="centerY" secondItem="Ze5-6b-2t3" secondAttribute="centerY" id="Wnf-iK-LGA"/>
                         </constraints>
-                        <viewLayoutGuide key="safeArea" id="6Tk-OE-BBY"/>
```

Comment:
these diffs from just opening the file in the Interface Builder always make me 🙄

Reply:
Very looking forward to the day we'll be past Interface Builder. Farewell old friend, you served us well back in the day, but haven't aged well 👋